lib/mutgen.c

```diff
@@ -146,41 +146,17 @@ mutation_matrix_print_state(mutation_model_t *self, FILE *out)
     fprintf(out, "\n");
 }
 
-static int MSP_WARN_UNUSED
-mutation_matrix_check_validity(mutation_matrix_t *self)
+static bool MSP_WARN_UNUSED
+ok_probability_masses(size_t num_alleles, double *distribution)
```

Comment:
to better resemble other functions we have, how about `validate_probabilities`? or `check_probabilities`?

Reply:
`ok_probability_masses` does seem to pretty much be another `check_validity` type function. And this `ok_` function is returning `bool` rather than returning zero for pass and non-zero for a specific error. So I'm thinking I should change the name and return type to be more consistent.